setup.py

```diff
@@ -51,12 +51,17 @@
 
 
 def here(*paths):
-    return os.path.abspath(os.path.join(os.path.dirname(__file__), *paths))
+    return os.path.relpath(os.path.join(*paths))
```

Comment:
`os.path.dirname(__file__)` will always be relative in this context, right?

On Sat, Jan 23, 2016 at 9:22 PM, Paul Kehrer <notifications@github.com>
wrote:

> In setup.py <https://github.com/pyca/pynacl/pull/171#discussion_r50629213>
> :
>
> > @@ -51,12 +51,17 @@
> >
> >
> >  def here(*paths):
> > -    return os.path.abspath(os.path.join(os.path.dirname(__file__), *paths))
> > +    return os.path.relpath(os.path.join(*paths))
>
> If we want here to join any path (even if it's already an absolute) and
> provide a path relative to the cwd then yes. Otherwise we should just
> remove here entirely? (Removing here entirely probably makes the most sense)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/pynacl/pull/171/files#r50629213>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084